core/src/main/scala/kafka/consumer/BaseConsumer.scala

```diff
@@ -40,6 +40,7 @@ trait BaseConsumer {
   def stop()
   def cleanup()
   def commit()
+  def resetOffsets(): Unit = {}
```

Comment:
Hmm.. I actually thought it was a bit nicer to keep this encapsulated in `NewShinyConsumer` since it is not needed for the old consumer. Was there a good reason to pull it up?